backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -269,98 +194,78 @@ public void addEmail(Email email) {
                 })
-                .onItem().transformToMultiAndConcatenate(result -> {
-                    if (delete) {
-                        return emailAggregationResources.purgeOldAggregation(aggregationKey, endTime)
-                                .toMulti()
-                                .onItem().transform(integer -> result);
-                    }
-
-                    return Multi.createFrom().item(result);
-                });
-                // Todo: If we want to save the NotificationHistory, this could be a good place to do so. We would probably require a special EndpointType
-                // .onItem().invoke(result -> { })
+                .onItem().ignoreAsUni();
     }
-
-    Uni<List<Tuple2<NotificationHistory, EmailAggregationKey>>> processAggregateEmails(Instant scheduledFireTime, EmailSubscriptionType emailSubscriptionType, boolean delete) {
-        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(emailSubscriptionType.getDuration());
-
-        LocalDateTime endTime = LocalDateTime.ofInstant(scheduledFireTime, UTC);
-        LocalDateTime startTime = LocalDateTime.ofInstant(yesterdayScheduledFireTime, UTC);
-        final LocalDateTime aggregateStarted = LocalDateTime.now();
-
-        log.info(String.format("Running %s email aggregation for period (%s, %s)", emailSubscriptionType.toString(), startTime.toString(), endTime.toString()));
-
-        return emailAggregationResources.getApplicationsWithPendingAggregation(startTime, endTime)
-                .onItem().transformToMulti(Multi.createFrom()::iterable)
-                .onItem().transformToMultiAndConcatenate(aggregationKey -> processAggregateEmailsByAggregationKey(aggregationKey, startTime, endTime, emailSubscriptionType, delete))
-                .collect().asList()
-                .onItem().invoke(result -> {
-                    final LocalDateTime aggregateFinished = LocalDateTime.now();
-                    log.info(
-                            String.format(
-                                    "Finished running %s email aggregation for period (%s, %s) after %d seconds. %d (accountIds, applications) pairs were processed",
-                                    emailSubscriptionType.toString(),
-                                    startTime.toString(),
-                                    endTime.toString(),
-                                    ChronoUnit.SECONDS.between(aggregateStarted, aggregateFinished),
-                                    result.size()
-                            )
-                    );
-                });
-    }
-
-    @Scheduled(identity = "dailyEmailProcessor", cron = "{email.subscription.daily.cron}")
-    public void processDailyEmail(ScheduledExecution se) {
-        // Only delete on the largest aggregate time frame. Currently daily.
-        processAggregateEmails(se.getScheduledFireTime(), EmailSubscriptionType.DAILY, true).await().indefinitely();
-    }
```

Comment:
If we merge this, the old way of sending the daily email will be gone. That means the new way _has to_ work and be reliable immediately. Otherwise, we won't be able to fix any production issue that would require to change and redeploy notifications-backend.

Shouldn't we keep the old way and migrate to the new way using a configuration property?

cc @josejulio @pilhuhn

Reply:
I would like to put it on stage first and check if it's working there. 
There is no plan to put it in prod immediately. But this approach could be a problem when someone else also wants to put something in prod in the meantime.

By using a feature toggle, this would be a no brainer. 

I think we should keep the old way, indeed.

Reply:
Yes, lets keep the old way meanwhile and use a flag (maybe a variable in properties?) to turn on/off (default on :D).

Reply:
Is "having it on status ON" a joke? Or do you want it to be on ON by default? :D

Reply:
I mean to have the "old way"  as default :D

Reply:
Okay. I disabled the "new way" with my last commit. Your desired behavior should be there now

Reply:
The [old scheduled job](https://github.com/bennetelli/notifications-backend/blob/7ec7cfc6126588c19a68c9459947c605df33db32/backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java#L288-L292) should be executed only if `notifications.aggregator.email.subscription.periodic.cron.enabled` is `true`.

Reply:
I am not sure if this would work. This entry is in another project and they are in different jars and have different application.properties. I would say this doesn't work.
We can add another property in notifcations-backend, but this somehow doesnt feel the right way

Reply:
Oh yes I forgot notifications was multi-module now, my bad 😄

I think adding the same config property in backend is fine as long as both properties (backend and aggregator) are commented with something explaining that they should always be modified in a synchronized way and the values should never be identical between both projects.

Reply:
I just added the property and comments in both application properties.